src/Client/Magento.php

```diff
@@ -133,6 +165,10 @@ public function getUrl(string $path, bool $async = false): string
             $options[] = 'async';
         }
 
+        if ($async && $bulk) {
```

Comment:
Could you modify this so that it only checks $bulk? 
In the async check above you may use `$async || $bulk`

I think that it will get confusing otherwise.

Reply:
Like that?

        if ($async || $bulk) {
            $options[] = 'async';

            if ($bulk) {
                $options[] = 'bulk';
            }
        }